Source/ExtendCustomModelType.swift

```diff
@@ -245,6 +245,10 @@ extension _ExtendCustomModelType {
                     continue
                 }
 
+                if mapper.propertyPresentNullValue(key: Int(bitPattern: info.address)) {
```

Comment:
Maybe we will use like this -> 
`useNullValueIfNeeded = mapper.propertyPresentNullValue(key: Int(bitPattern: info.address))` 

just a proposal